src/cheri/riscv-priv-integration.adoc

```diff
@@ -665,3 +665,160 @@ compatible with RISC-V harts provided that <<pcc>> and <<ddc>> are set to
 ==== Memory Accesses
 
 If the effective address of the memory access is invalid, and the authorizing capability's bounds do not cover all addresses, then raise a {cheri_excep_name_ld} or {cheri_excep_name_st} exception because the bounds cannot be reliably decoded.
+
+==== Glossary
```

Comment:
Given the common refrain of "see the unprivileged manual", perhaps this is more a "list of terms defined in the unprivileged manual" rather than a "Glossary" per se?

Reply:
yes good point!